doc/devel/documenting_mpl.rst

```diff
@@ -341,6 +342,14 @@ Note that the python script that generates the plot is referred to, rather than
 any plot that is created.  Sphinx-gallery will provide the correct reference
 when the documentation is built.
 
+Writing mathematical expressions
+--------------------------------
+
+In most cases, you will likely want to use one of `Sphinx's builtin Math
```

Comment:
Somewhat disagree with this.  We don't want to confuse people, and 99.9% of math should be written using the native sphinx extensions so this should be the first sentence.

Reply:
I think 99% of people landed on this doc because they want to write our docs and, in this specific subsection, write math in our docs.  _We_ landed here to clarify the difference, but that is not what most people care about, they just want to be told what to do. Agree about table setting in the actual doc that discusses our math extension.

Reply:
How about this: calling it "Tools for writing mathematical expressions" sets the tone for this being about different ways to write expressions and when to choose which. Does that make it better?